utilities/CMakeLists.txt

```diff
@@ -23,23 +23,23 @@ project (utilities)
 
 list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/../amd_openvx/cmake)
 
-find_package(OpenCV QUIET)
 find_package(OpenCL QUIET)
 find_package(FFmpeg QUIET)
 
-set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
-set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
-set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
-set(CMAKE_INSTALL_PREFIX /opt/rocm/mivisionx)
-
 if(OpenCL_FOUND)
     add_subdirectory (runvx)
     add_subdirectory (runcl)
     add_subdirectory (loom_shell)
     if(FFMPEG_FOUND)
         add_subdirectory (mv_deploy)
+        message("-- MIVisionX Utilities - RunVX, RunCL, Loom Shell, & MV Deploy modules added")
         install (FILES ./mv_deploy/mvdeploy_api.h DESTINATION include)
         install (FILES ./mv_deploy/mv_extras_postproc.h DESTINATION model_compiler)
         install (FILES ./mv_deploy/mv_extras_postproc.cpp DESTINATION model_compiler)
+    else()
+        message("-- MIVisionX Utilities - RunVX, RunCL, & Loom Shell modules added")
+        message("-- FFMPEG Not Found - MV Deploy module excluded")
```

Comment:
better to give the name as in the install director (runvx, runcl, loom_shell, mv_deploy)